thunderbird/processes/wps_generate_climos.py

```diff
@@ -139,7 +139,8 @@ def dry_run_info(self, filepath, climo):
             f.write("Dry Run\n")
             logging.basicConfig(filename=filename, level=logging.INFO)
             dry_run_handler(filepath, climo)
-            logging.basicConfig(stream=sys.stderr, level=logging.INFO)
```

Comment:
Since you removed this statement, you can remove `import sys` at the top.

Reply:
Good point! I totally forgot about that.